File: c/trpm-s1/TD1-ex-9-a.c
#include <stdio.h>
int addition(int nomber);

int main() {
  int N, sum = 0;
  printf("Donner la valeur de N \n");
  scanf("%d", &N);

  sum = addition(N);
  printf("la sum egal %d\n", sum);

  return 0;
}

int addition(int nomber) {
  int resultat = 0;
  for (int i = 0; i <= nomber; i++) {
    resultat = resultat + i;
  }
  return resultat;
}

File: matlab/trpm-s1/TD1-ex-9-a.m
function resultat = addition(nomber)
  resultat = 0;
  for i = 0:nomber
    resultat = resultat + i;
  end
endfunction

N=input("Donner la valeur de a \n");
sum = addition(N);
printf("la sum egal %d\n", sum);
